project/__init__.py

```diff
@@ -0,0 +1,60 @@
+import os.path
+from glob import glob
+
+ROOTDIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+TEMPLATEDIR = os.path.join(ROOTDIR, 'templates')
+
+
+class BaseProject(object):
+
+    def __init__(self, name):
+        self.name = name
+        self.builddir = os.path.join(TEMPLATEDIR, self.name, 'build/')
+
+    @property
+    def cleanup(self):
+        return ['rm -rf {0}'.format(self.builddir)]
+
+    @property
+    def init(self):
+        if hasattr(self, 'github_name'):
+            name = self.github_name
+        else:
+            name = self.name
+        return ['git clone git@github.com:platformsh/template-{0}.git {1}'.format(
+            name, self.builddir)
+        ]
+
+    @property
+    def platformify(self):
+        actions = ['rsync -aP {0} {1}'.format(
+            os.path.join(TEMPLATEDIR, self.name, 'files'),  self.builddir
+        )]
```

Comment:
This appears to be rsyncing from the wrong source.  See: https://github.com/platformsh/template-drupal8/pull/23/files

It looks like the `files` directory is included in the content that gets synced.  That probably means we need a `/` on the `files` string in the join, because rsync is super picky about its slashes.

Reply:
I confirmed in testing that just adding a / seems to do the job.  I think. :smile:

Reply:
Fixed, resulting actions list for platformify command now looks the same as in the master branch.